sdk/communication/azure-communication-rooms/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## 1.0.0-alpha.1 (2021-12-17)
```

Comment:
We generally only use `beta`. Replace `alpha` with `beta` in all version references.